testsuite/tests/frame-pointers/unhandled_effects.ml

```diff
@@ -0,0 +1,37 @@
+(* TEST
+ frame_pointers;
```

Comment:
do we really want to restrict the test to only setups with frame pointers enabled? The current fix is for a bug that only occurs with frame pointers, but I assume that the test may have caught the non-frame-pointers-specific s390x issue, and could be helpful in the test harness of new backends.

Reply:
(To clarify: my gut feeling is that we do *not* want the `frame_pointers` condition here, and I am waiting to see what @dustanddreams thinks before merging, with or without the condition.)

Reply:
Yes, you're right; although the code was only incorrect with frame pointers enabled, this was not the case on s390x. But then, should it really belong to the `tests/frame-pointers` directory?

Reply:
> But then, should it really belong to the tests/frame-pointers directory?

Honestly I don't really care, please choose the test directory that you prefer.

Reply:
Moved to `effects`, then.